language/Type-Classes.md

```diff
@@ -137,36 +137,128 @@ See also the section in [PureScript by Example](https://book.purescript.org/chap
 
 ## Type Class Deriving
 
-Some type class instances can be derived automatically by the PureScript compiler. To derive a type class instance, use the `derive instance` keywords:
+The compiler can derive type class instances to spare you the tedium of writing boilerplate. There are a few ways to do this depending on the specific Type and Class being derived.
 
-```purescript
-newtype Person = Person { name :: String, age :: Int }
+### Derive from `newtype`
+
+If you'd like to use a wrapped type's instance for your newtype, use the `derive newtype` keywords.
```

Comment:
I'm not sure this fully captures the idea that you're deferring to the wrapped type's instance -- that it's almost like your newtype is ignored and the instance goes directly to whatever the wrapped type does for that class.

What about this tweak?

> If you would like your newtype to defer to the instance that the underlying type uses for a given class, then you can use newtype deriving via the `derive newtype` keywords.